integration-tests/contracts/ethereum_keeper_contracts.go

```diff
@@ -248,9 +253,25 @@ func (rcs *KeeperRegistrySettings) EncodeOnChainConfig(registrar string, registr
 			UpkeepPrivilegeManager: registryOwnerAddress,
 		}
 
-		encodedOnchainConfig, err := utilsABI.Methods["_onChainConfig"].Inputs.Pack(&onchainConfigStruct)
+		encodedOnchainConfig, err := utilsABI21.Methods["_onChainConfig"].Inputs.Pack(&onchainConfigStruct)
 
 		return encodedOnchainConfig, err
+	} else if rcs.RegistryVersion == ethereum.RegistryVersion_2_2 {
+		chainModuleBaseAddr, _, _, err := client.DeployContract("ChainModuleBase", func(
```

Comment:
@FelixFan1992 can we have `ChainModuleBase` contract deployment part inside contract_deployer.go, inside `DeployKeeperRegistry`? That will let us avoid passing the client to this function.

Reply:
fixed